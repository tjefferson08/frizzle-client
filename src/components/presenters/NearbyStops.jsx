import React from 'react';
import StopsNearMeButton from '../containers/StopsNearMeButton';
import VisibleStopList from '../containers/VisibleStopList';
import MapContainer from '../containers/MapContainer';
import ToggleMapButton from '../containers/ToggleMapButton';

const NearbyStops = () => {
  return (
    <div>
      <span>
        Stops near:
      </span>
      <StopsNearMeButton text="My current location" />
      <ToggleMapButton text="Toggle map"/>
      <MapContainer showMap={true} />
      <VisibleStopList />
    </div>
  );
};

export default NearbyStops;

