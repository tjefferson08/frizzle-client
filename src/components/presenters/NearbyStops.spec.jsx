import React from 'react';
import ReactTestUtils from 'react-addons-test-utils';
import NearbyStops from './NearbyStops';

test('should render a li', () => {
  const renderer = ReactTestUtils.createRenderer();
  renderer.render(<NearbyStops />)
  const result = renderer.getRenderOutput();

  expect(result.type).toBe('div');
  expect(result).toMatchSnapshot();
});

