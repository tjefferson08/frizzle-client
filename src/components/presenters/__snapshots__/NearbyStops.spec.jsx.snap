exports[`test should render a li 1`] = `
<div>
  <span>
    Stops near:
  </span>
  <Connect(Button)
    text="My current location" />
  <Connect(Button)
    text="Toggle map" />
  <Connect(MapContainer)
    showMap={true} />
  <Connect(StopList) />
</div>
`;
